openscm/core/__init__.py

```diff
@@ -80,7 +80,8 @@ def output(self) -> ParameterSet:
         """
         Set of output parameters of the model
         """
-        return self._output_parameters
+        raise NotImplementedError  # noqa
```

Comment:
Why mark that as not implemented? It's just a getter...

Reply:
same below...

Reply:
because it's not being tested, again just to make sure that you've seen the line